### ISSUE TITLE
Is it worth keeping this repo up to date (alive) ?

### ISSUE THREAD

**Issue by viperadnan-git** [viperadnan-git/google-drive-telegram-bot#99]
As per the new policy of Google Workspace, all unauthorized Team Drives are going to be terminated soon. So ultimately it's gonna effect the data hoardings (which is a good thing) this repo aims to work with Google Drive but when i created this i thought of making some helpful bot for data hoarders. Should i keep this repo alive or keep working on other projects ? Or should i use rclone instead to support multiple cloud storage ?

Drop your opinions...

**Comment by SonebhadraAutoComapny** [comment]
rclone with ability to add other cloud providers will be even more cooler imo

**Comment by Akash23q2** [comment]
rclone with ability to add more cloud providers will be even more cooler imo

**Comment by abhishek-sharma8** [comment]
Rclone sounds like a great idea.
Thanks for the bot.

**Comment by ghost** [comment]
even if team drives get terminated or discontinues we can always use  personal 15 gb storage also rclone idea  seems cool

**Comment by cosmicqbit** [comment]
Of course, this repo still stays relevant despite the amendment in the Google Workspace policy. Because that will put only unauthorized Shared Drives in danger, those who still manage to have their Shared Drives running whether with their legal subscription or freely generated, this repo is going to serve them for sure. 

Also thanks for the uploader 😄

**Comment by ghost** [comment]
yeah please continue working many of us have an enterprise subscriptions some also get storage from school/ college so add rclone please

**Comment by BotTutorials** [comment]
#118
